clients/internal/pom.xml

```diff
@@ -0,0 +1,49 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.tachyonproject</groupId>
+    <artifactId>tachyon-clients</artifactId>
+    <version>0.7.0-SNAPSHOT</version>
+  </parent>
+  <groupId>org.tachyonproject</groupId>
+  <artifactId>tachyon-client-internal</artifactId>
```

Comment:
Maybe we could call this `unshaded` instead of `internal` since this jar isn't restricted to internal use?